.aws/src/main.ts

```diff
@@ -332,7 +322,7 @@ class RecommendationAPI extends TerraformStack {
             },
 
             autoscalingConfig: {
-                targetMinCapacity: config.environment === 'Prod' ? 4 : 4,
```

Comment:
do you want to keep it at least 2. Not sure what was previous minimum

Reply:
Our guideline is to set it to 1 in the dev environment to reduce cost and emissions. For some reason that wasn't applied here.